blockchain/chain_sync/src/sync.rs

```diff
@@ -418,6 +418,16 @@ where
             // Currently needed to go GT because equal tipsets are attempted to be synced.
             .map(|heaviest| ts.weight() > heaviest.weight())
             .unwrap_or(true);
+        info!(
+            "New tipset heavier than heaviest: {}, and heaviest weight: {:?}, new weight: {}",
+            candidate_ts,
+            self.state_manager
+                .chain_store()
+                .heaviest_tipset()
+                .await
+                .map(|e| e.weight().clone()),
+            ts.weight()
+        );
```

Comment:
do we want this log? or was this just added for debugging? I don't see the benefit in this being logged